Cargo.toml

```diff
@@ -37,6 +37,7 @@ bytecount = "0.6.3"
 clap.workspace = true
 clap_complete = "4.3.2"
 crossbeam-channel = "0.5.8"
+gengo = "^0.5.0"
```

Comment:
Now that `gengo` is also using `gix` it's worth noting that both need to coordinate their `gix` versions while pre-1.0. Otherwise you would end up with two distinct `gix`-dependency trees which costs double to compile and in the final binary.

Reply:
I think we would want to create a group in the dependabot config so that `gix` and `gengo` are updated simultaneously. On gengo's side I can try to promptly create a new release for significant `gix` updates.